apps/client/app/routes/_layout.emails.tsx

```diff
@@ -0,0 +1,114 @@
+import { Form, useLoaderData, useNavigate, useNavigation, useSubmit } from "@remix-run/react";
```

Comment:
For these emails it might be better for readability to separate routes by folders inside /routes e.g:
for email related '/routes/email/' and so on.

Reply:
Yeah Remix also recommends this for scaling but beware that this [changes route behaviour a bit](https://remix.run/docs/en/main/file-conventions/routes#folders-for-organization).

Reply:
> For these emails it might be better for readability to separate routes by folders inside /routes e.g: for email related '/routes/email/' and so on.

Ok so, I took at the documentation and Im not sure if changing the routes to a folder base structure is the right approach. Even with folders we cannot nest folders within folders. We would still need to have a folder for each route even if a route is nested within another.

For example:

routes/_layout.tsx
routes/_layout.emails.tsx
routes/_layout.emails_.$emialId.tsx

will result in 3 different folders:

routes/_layout/route.tsx
routes/_layout.emails/route.tsx
routes/_layout.emails_.$emialId/route.tsx

![image](https://github.com/diconium/ethereal-pulse/assets/174435266/535c3904-263f-4f44-802d-5376060508c9)

Regarding scaling, I think the advantage that folder base routes have is that you can place inside each route folder specific components exclusively used by that route page.

Reply:
then I would say we can skip this for now and update if we see the need